src/components/common/Footer.tsx

```diff
@@ -1,100 +1,96 @@
+                <h3 className="font-semibold text-gray-400">HOME</h3>
+                <ul className="mt-2 space-y-2">
+                  <li className="border-b border-gray-600 pb-1">
+                    <Link href="/about">About Me</Link>
+                  </li>
+                  <li className="border-b border-gray-600 pb-1">
+                    <Link href="/works">My Works</Link>
+                  </li>
+                  <li className="border-b border-gray-600 pb-1">
+                    <Link href="/testimonials">Testimonials</Link>
+                  </li>
+                </ul>
+              </div>
+              <div className="w-1/2 sm:w-1/4 lg:w-auto mb-6 lg:mb-0">
+                <h3 className="font-semibold text-gray-600">CLIENTS</h3>
+                <ul className="mt-2 space-y-2">
+                  <li className="border-b border-gray-600 pb-1">Klovesto</li>
+                  <li className="border-b border-gray-600 pb-1">Nukeway</li>
+                  <li className="border-b border-gray-600 pb-1">Cloven's</li>
+                  <li className="border-b border-gray-600 pb-1">Menvol</li>
+                </ul>
+              </div>
+              <div className="w-1/2 sm:w-1/4 lg:w-auto mb-6 lg:mb-0">
+                <h3 className="font-semibold text-gray-600">PORTFOLIO</h3>
+                <ul className="mt-2 space-y-2">
+                  <li className="border-b border-gray-600 pb-1">Events</li>
+                  <li className="border-b border-gray-600 pb-1">Portrait</li>
+                  <li className="border-b border-gray-600 pb-1">Branding</li>
+                  <li className="border-b border-gray-600 pb-1">Commerciale</li>
+                  <li className="border-b border-gray-600 pb-1">Wedding</li>
+                </ul>
+              </div>
+              <div className="w-1/2 sm:w-1/4 lg:w-auto mb-6 lg:mb-0">
+                <h3 className="font-semibold text-gray-600">SERVICES</h3>
+                <ul className="mt-2 space-y-2">
+                  <li className="border-b border-gray-600 pb-1">Portraits</li>
+                  <li className="border-b border-gray-600 pb-1">Events</li>
+                  <li className="border-b border-gray-600 pb-1">Commercial</li>
+                </ul>
+              </div>
             </div>
-            <div>
-              <h3 className="font-semibold">CLIENTS</h3>
-              <ul className="mt-2 space-y-2">
-                <li>Klovesto</li>
-                <li>Nukeway</li>
-                <li>Cloven's</li>
-                <li>Menvol</li>
-              </ul>
+          </div>
+        </div>
+        <div className="border-t border-gray-600 py-8">
+          <div className="container mx-auto px-4 flex flex-col md:flex-row justify-between items-center">
+            <div className="text-sm mb-4 md:mb-0">
+              <Link href="/terms">Terms & Conditions</Link> |{" "}
+              <Link href="/privacy">Privacy Policy</Link>
```

Comment:
```suggestion
              <Link href="/terms">Terms & Conditions</Link>
              {" | "}
              <Link href="/privacy">Privacy Policy</Link>
```
I think this one looks a bit better